### ISSUE TITLE
iOS Sample NullReferenceException occured by start accelerometer 

### ISSUE THREAD

**Issue by matatabi-ux** [mattleibow/Microsoft-Band-SDK-Bindings#3]
NullReferenceException occured by iOS Sample demo app.

Exception occured BandException.cs at 30 line.

```
public BandException (NSError error)
    : this (error.LocalizedDescription, error)
{
}
```

'error' is nullable, so I think that 'error.LocalizedDescription' occur a  NullReferenceException.
This exception was avoid in the below code.

```
public BandException (NSError error)
    : this (error != null ? error.LocalizedDescription : string.Empty, error)
{
}
```

**Comment by mattleibow** [comment]
@matatabi-ux Thanks for this. I decided to fix the actual cause in commit 26263385d3327b5146b03660cde591fab711583c instead of the exception as passing a null error results in an empty exception. If the result was a success, we shouldn't be creating an exception at all.

**Comment by mattleibow** [comment]
Fixed in commit 26263385d3327b5146b03660cde591fab711583c
